packages/page/src/UserInterface/Command/InitializeHomepageCommand.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Page\UserInterface\Command;
+
+use Sulu\Component\Webspace\Manager\WebspaceManagerInterface;
+use Sulu\Component\Webspace\Webspace;
+use Sulu\Content\Domain\Model\WorkflowInterface;
+use Sulu\Messenger\Infrastructure\Symfony\Messenger\FlushMiddleware\EnableFlushStamp;
+use Sulu\Page\Application\Message\ApplyWorkflowTransitionPageMessage;
+use Sulu\Page\Application\Message\CreatePageMessage;
+use Sulu\Page\Application\MessageHandler\CreatePageMessageHandler;
+use Sulu\Page\Domain\Model\PageInterface;
+use Sulu\Page\Domain\Repository\PageRepositoryInterface;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Messenger\Envelope;
+use Symfony\Component\Messenger\HandleTrait;
+use Symfony\Component\Messenger\MessageBusInterface;
+
+/**
+ * @internal Your code should not create direct dependencies on this implementation
+ *  projects can utilize the `sulu:page:initialize` command to
+ *  initialize new webspaces with the homepage
+ *
+ * @final
+ */
+#[AsCommand(name: 'sulu:page:initialize', description: 'Initializes the homepage per webspace')]
+class InitializeHomepageCommand extends Command
+{
+    use HandleTrait;
+
+    public function __construct(
+        private WebspaceManagerInterface $webspaceManager,
+        private PageRepositoryInterface $pageRepository,
+        MessageBusInterface $messageBus
+    ) {
+        parent::__construct();
+        $this->messageBus = $messageBus;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $ui = new SymfonyStyle($input, $output);
+
+        $webspaces = $this->webspaceManager->getWebspaceCollection();
+        foreach ($webspaces as $webspace) {
+            $ui->section('Initialize homepage for webspace ' . $webspace->getKey());
```

Comment:
I don't know how well Symfony Style and verbosity are but I think we shouldn't print all the output to the console. Maybe we could only print like a high level overview and then with `-v` for more details.

Reply:
I’ve removed this line entirely, as I don’t think it’s necessary. Only the information that a homepage already exists for specific webspace/locale combinations is now being logged.